test/test.js

```diff
@@ -172,3 +172,54 @@ describe('.within(start,finish)', function () {
     after: 'assert(two < five || ten < two)'
   });
 });
+
+describe('.throw([errorLike], [errMsgMatcher])', function () {
```

Comment:
signature of expect.js's `throws` is different from chai's

- chai: `.throw([errorLike], [errMsgMatcher])`
- expect.js: `.throw([Function|RegExp])`

Therefore chai's testcase does not fit. You should modify them for expect.js (That's why I created `expect-js-to-assert`)

```js
describe('.throw([Function|RegExp])', function () {
  testTransform({
    before: 'expect(func).to.throw()',
    after: 'assert.throws(func)'
  });
  testTransform({
    before: 'expect(func).to.not.throw()',
    after: 'assert.doesNotThrow(func)'
  });
  testTransform({
    before: 'expect(func).to.throw(/error message/)',
    after: 'assert.throws(func, /error message/)'
  });
  testTransform({
    before: 'expect(func).to.not.throw(/error message/)',
    after: 'assert.doesNotThrow(func, /error message/)'
  });
  testTransform({
    before: 'expect(func).to.throw(handler)',
    after: 'assert.throws(func, handler)'
  });
  testTransform({
    before: 'expect(func).to.not.throw(handler)',
    after: 'assert.throws(func, handler)'
  });
  testTransform({
    before: 'expect(func).to.throwError()',
    after: 'assert.throws(func)'
  });
  testTransform({
    before: 'expect(func).to.not.throwError()',
    after: 'assert.doesNotThrow(func)'
  });
  testTransform({
    before: 'expect(func).to.throwException()',
    after: 'assert.throws(func)'
  });
  testTransform({
    before: 'expect(func).to.not.throwException()',
    after: 'assert.doesNotThrow(func)'
  });
});
```